odxtools/table.py

```diff
@@ -56,6 +56,38 @@ def __post_init__(self) -> None:
         self._structure: Optional[DopBase] = None
         self._dop: Optional[DopBase] = None
 
+    @staticmethod
+    def from_et(et_element, doc_frags: List[OdxDocFragment]) \
+            -> "TableRow":
+        """Reads a TABLE-ROW."""
+        odx_id=OdxLinkId.from_et(et_element, doc_frags)
+        assert odx_id is not None
+        short_name=et_element.findtext("SHORT-NAME")
```

Comment:
You need a code style linter
```suggestion
        short_name = et_element.findtext("SHORT-NAME")
```

Reply:
I don't mind. but let's not mingle these changes with functional ones or this PR will become even more unreviewable than it already is, i.e., let's do a separate giant PR which just puts the whole code base through one of the linters (black?)

Reply:
sounds good

Reply:
Black is a nice formatter, which is widely used in OS projects. The major drawback is, that it strictly limits lines to roughly 90 characters. Especially with the verbose typing annotations, this will result in some cumbersome method signatures or variable declarations.

I agree, this should be part of a different PR, since black would probably reformat the entire codebase and the functional changes would be untraceable.